src/main/java/neo/MarkCommand.java

```diff
@@ -21,5 +21,9 @@ void complete(String tempi) throws NeoException, IOException {
         arrayLL.getTask(tempii-1).setIsDone(true);
         System.out.println("Nice! I've marked this task as done");
         System.out.println(arrayLL.getTask(tempii-1).toString());
+        stor.Writedata(arrayLL.getTask(0));
+        for(int i =1; i<arrayLL.arrayL.size(); i++) {
```

Comment:
Convention to have a space around operators based on Java coding standard, this line of code should be written as:
`for (int i = 1; i < arrayLL.arrayL.size(); i++) {`